core-ui/src/components/Extensibility/errors.md

```diff
@@ -0,0 +1,34 @@
+### Extensibility Error Handling
+
+There's a special error handler, that can be used when validating input from the EXT Config Maps.
+
+Users make errors in configuration, we can display the error's message and even the responsible part of a Config Map with the `throwConfigError` handler:
+
+```js
+if (!structure || typeof structure !== 'object') {
```

Comment:
I don't think we need to fully explain what will be displayed, which cases are handled here, show our code, etc. I don't even think we need separate documentation for that (since the user cannot do anything about it, it's not being configured or anything). We can have a short mention in the main documentation, that would be enough IMHO.

Reply:
It wasn't meant for the users, but for the developers who work on the software. A beginning to a guide 'how to work with extensibility'.

Reply:
Then @grego952 could you suggest what would be the best place for such docs? Should we keep it in the same place as docs for the users or should we move it somewhere else?

Reply:
Correct me if I'm wrong but I think that most Busola users are developers. If it's only a short mention, I'd keep it in the main documentation.

Reply:
Or if it's only for our team, maybe we can put it in our internal repo?